Sources/MapboxNavigation/OrnamentsController.swift

```diff
@@ -171,14 +171,14 @@ extension NavigationMapView {
             if mapView.streetsSources().isEmpty {
                 var streetsSource = VectorSource()
                 streetsSource.url = "mapbox://mapbox.mapbox-streets-v8"
-                mapView.style.addSource(source: streetsSource, identifier: "com.mapbox.MapboxStreets")
+                try? mapView.style.addSource(streetsSource, id: "com.mapbox.MapboxStreets")
```

Comment:
Here’s an instance in which we’re swallowing an error without even logging anything. Should we handle runtime styling failures more consistently?

Reply:
Thanks, missed that one. Will add error handling.

Reply:
Also for cases when searching for source/layer I did not add error handling logging because it's not necessarily an error when source/layer is not present.